respa_exchange/downloader.py

```diff
@@ -245,8 +292,11 @@ def fetch_reservation_data(ex_reservation):
     )
     session = ex_reservation.exchange.get_ews_session()
     items = [item for item in gcir.send(session)]
+    if len(items) == 0:
+        return None
+
     assert len(items) == 1, "Exchange returned %d items instead of 1" % (len(items))
-    return str(etree.tostring(items[0], pretty_print=True), encoding='utf8')
+    return items[0]
```

Comment:
Hmm, cannot find any usages of this function?

Reply:
It is used in some auxiliary scripts that are not stored in the repo.